atomics/T1045/T1045.yaml

```diff
@@ -0,0 +1,96 @@
+---
+attack_technique: T1045
+display_name: Software Packing
+
+atomic_tests:
+- name: Binary simply packed by UPX (linux)
+  description: |
+    Copies and then runs a simple binary (just outputting "the cake is a lie"), that was packed by UPX.
+    No other protection/compression were applied.
+
+  supported_platforms:
+    - linux
+
+  input_arguments:
+    bin_path:
+      description: Packed binary
+      type: Path
+      default: bin/linux/test_upx
```

Comment:
Could you use the magic "PathToAtomicsFolder" variable here which give the framework the location of the atomics folder containing all the T# folder no matter where the user executes the test from. So it would look like "PathToAtomicsFolder/T1045/bin/linux/test_upx"  (Applies to following test input args as well)

Reply:
fixed in 8380d43